tests/lib/bin/session-tool

```diff
@@ -112,6 +121,18 @@ cat "$tmp_dir/ready.pipe" >/dev/null
 cat >"$tmp_dir/stdin.pipe" &
 cat <"$tmp_dir/stdout.pipe" >&1 &
 cat <"$tmp_dir/stderr.pipe" >&2 &
+(
+	echo "#!/bin/sh"
+	test -n "$pid_file" && echo "echo \$$ >\"$pid_file\""
```

Comment:
would it be possible to set a PIDFile property on the unit?

Reply:
IMO that is not useful as the unit runs runuser, which spawns the real workload.